services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -391,13 +403,14 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
               />
             )}
             {isEditMode && (
-              <ReportSubmissions
+              <ReportFileUpload
```

Comment:
It's showing as a completely new file, but I changed ReportSubmissions --> ReportFileUpload, since we are only dealing with one submission and don't really need to parse its documents (much simpler here at least)